pyfstat/mcmc_based_searches.py

```diff
@@ -998,14 +998,21 @@ def plot_chainconsumer(self, save_fig=True, label_offset=0.25, dpi=300, **kwargs
             )
             return
 
-        if "truths" in kwargs and len(kwargs["truths"]) != self.ndim:
```

Comment:
From a pure PyFstat user perspective, it would be nicer to keep calling the method argument called `truths` and internally change it to `truth` before calling `chainconsumer`. On the other hand, users already familiar with `chainconsumer` from the outside world would find that baffling. So let's go with this choice for now, and there's already #160 for general unification later on.